lib/midi.coffee

```diff
@@ -8,19 +8,36 @@ L = utils = midi = midi_in = ___ = null
   midi = rtMidi
   ___ = utils.____ '[midi]'
 
+# Parse method copied from https://github.com/hhromic/midi-utils-js/blob/master/midiparser.js#L418
```

Comment:
I think linking to the lines of code is sufficient too. Although, we could pull in this whole parser and use it if we wanted to.